beesdoo_product_label/rename_beesdoo.py

```diff
@@ -69,8 +69,6 @@ def rename_beesdoo(cr):
         """
         UPDATE ir_model_data SET module = %s
         WHERE module = %s AND
-        WHERE name = 'model_beesdoo_product_label' AND
-        name NOT IN
-        (SELECT name FROM ir_model_data WHERE module = %s)""",
-        (NEW_MODULE_NAME, OLD_MODULE_NAME, NEW_MODULE_NAME),
+        name LIKE 'model_beesdoo_product_label%'""",
+        (NEW_MODULE_NAME, OLD_MODULE_NAME),
```

Comment:
This query breaks when I run update-all on oufticoop.

```
2022-12-20 10:13:53,360 492218 ERROR 12-oufticoop odoo.sql_db: bad query: 
        UPDATE ir_model_data SET module = %s
        WHERE module = %s AND
        name LIKE 'model_beesdoo_product_label%'
ERROR: tuple index out of range 
2022-12-20 10:13:53,365 492218 WARNING 12-oufticoop odoo.modules.loading: Transient module states were reset 
2022-12-20 10:13:53,373 492218 ERROR 12-oufticoop odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/modules/loading.py", line 421, in load_modules
    loaded_modules, update_module, models_to_check)
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/modules/loading.py", line 313, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/modules/loading.py", line 186, in load_module_graph
    getattr(py_module, pre_init)(cr)
  File "/var/home/coopiteasy/cie/odoo12/src/coopiteasy/obeesdoo/beesdoo_product_label/rename_beesdoo.py", line 73, in rename_beesdoo
    (NEW_MODULE_NAME, OLD_MODULE_NAME),
  File "/var/home/coopiteasy/.virtualenvs/odoo12/lib64/python3.7/site-packages/openupgradelib/openupgrade.py", line 1481, in logged_query
    cr.execute(query, args)
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/var/home/coopiteasy/cie/odoo12/src/odoo/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
IndexError: tuple index out of range
```

I don't really understand why.

Furthermore, I suspect that `name LIKE '%beesdoo_product_label%'` is more accurate.

Reply:
Or rather, `name LIKE '%%product_label%%'`, because it should already have been renamed by the rename command earlier.

And then re-add the `name NOT IN (SELECT name FROM ir_model_data WHERE module = %s)` check.

Sorry for the spam.